drafts/interaction-guide.md

```diff
@@ -0,0 +1,73 @@
+
+**This list of recommendations was provided to volunteers for the 
+NumFOCUS booth at AAS 235.**
+
+The code of conduct sets the ground state for professionalism, but
+there are many ways to inadvertently cause harm during interactions
```

Comment:
I was not at a AAS, but this wording seems to imply that it's really dangerous to be at NumFOCUS booth. I would not like to volunteer there, because there are so many ways I can cause harm. Maybe I better don't do anything!

To avoid that impression, I suggest to tone this down a little: Just deleting the second have of the sentence "but ..." would work.

Reply:
Actually, @hamogu , I think that's not too far off from what we're getting at. We've had a couple of cases where one interaction has really hurt someone's feelings and turned them off from the Project entirely. We want to be sure people who are in "public-facing" positions understand how easy this is to do and to put significant effort into doing no harm. I think we just need to be more clear about what we mean by "harm". 

How about, "There are many ways to inadvertently insult or condescend to our users."

Reply:
If you want to be that explicit, because you experienced people at the booth doing real damage, then maybe it has to be the way you originally suggested.

I'm just saying, that, if you want to recruit me as a volunteer to run a booth, you'll have a hard time to convince me if it starts out with pointing out that there are so many ways I can harm the cause.

I'd prefer a more gentle wording in the written policy and a verbal explanation of what to look out for. Written policies get read without context - but maybe the bottom line is that booth like that should not be staffed by random people, but only by per-screened professionals who have proven that they handle the outreach efforts appropriately (but if that's that case, then such a policy would not be needed).

Either way, I was not there, I'm just reading the PR here and raising the flag "this policy would not want me to volunteer time". If you want to be gentle, don't include "there are many ways to insult", instead say "User may feel itimided or insulted easily, so ...". If you think that a *gentle* policy is not enough, go for your original text, but know that you are turning away some fraction of potential volunteers in the future.